NuRadioMC/SignalProp/examples/birefringence_examples/05_Veff_comparison.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import absolute_import, division, print_function
```

Comment:
how long does it take to execute this example?

Reply:
about 90s on my laptop